src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -89,7 +89,7 @@ class Submitter(stagingBaseDir: File,
       stageDir = (stagingBaseDir / draftDeposit.id.toString).createDirectories()
       stageBag <- DansV0Bag.empty(stageDir / "bag").map(_.withCreated())
       // EASY-1464 3.3.6 change state and copy with the rest of the deposit properties to staged dir
-      submittedId <- draftDeposit.setStateInfo(StateInfo(StateInfo.State.submitted, "Deposit is ready for processing."))
+      submittedId <- stateManager.changeState(StateInfo(StateInfo.State.submitted, "Deposit is ready for processing."))
```

Comment:
Beyond the scope of this issue. 

We want to copy the new state to the `submitDir` but the new state should be saved in the draft when successfully moved moved from the staged to submitted directory.